### ISSUE TITLE
Error when using jsdom and analyzing local version of https://edge.ms

### ISSUE THREAD

**Issue by melanierichards** [sonarwhal/sonar#601]
## Environment

* __`sonar` version: 0.12.3
* __`Node.js` version: 8.8.1
* __`npm` version: 5.4.2

## `sonar` configuration

<details>
<summary>sonar's configuration</summary>

```js

{
    "browserslist": [],
    "connector": {
        "name": "jsdom",
        "options": {
            "waitFor": 1000
        }
    },
    "formatters": [
        "json"
    ],
    "ignoredUrls": {},
    "rules": {
        "amp-validator": "off",
        "apple-touch-icons": "error",
        "axe": "error",
        "content-type": "error",
        "disown-opener": "error",
        "highest-available-document-mode": "error",
        "html-checker": "error",
        "image-optimization-cloudinary": "off",
        "manifest-app-name": "error",
        "manifest-exists": "error",
        "manifest-file-extension": "error",
        "manifest-is-valid": "error",
        "meta-charset-utf-8": "error",
        "meta-viewport": "error",
        "no-disallowed-headers": "error",
        "no-friendly-error-pages": "error",
        "no-html-only-headers": "error",
        "no-protocol-relative-urls": "error",
        "no-vulnerable-javascript-libraries": "error",
        "ssllabs": "off",
        "strict-transport-security": "error",
        "validate-set-cookie-header": "error",
        "x-content-type-options": "error"
    },
    "rulesTimeout": 120000
}

```

</details>

## URL for which `sonar` failed

* __URL: http://localhost:3000

## Output

Please include the raw output generated by `sonar`, or if possible,
the raw output from running `sonar` in debug mode: `sonar --debug <URL>`.

<details>
<summary>sonar's raw output</summary>

```text

\ Downloading https://uhf.microsoft.com/_log?o=mscc&m=showTypeError: Cannot read property 'name' of null
    at validate (/usr/local/lib/node_modules/@sonarwhal/sonar/dist/src/lib/rules/manifest-app-name/manifest-app-name.js:54:38)
    at Immediate.setImmediate [as _onImmediate] (/usr/local/lib/node_modules/@sonarwhal/sonar/dist/src/lib/sonar.js:144:46)
    at runCallback (timers.js:785:20)
    at tryOnImmediate (timers.js:747:5)
    at processImmediate [as _immediateCallback] (timers.js:718:5)

```

</details>

**Comment by molant** [comment]
The site being analyze is https://edge.ms when running locally.

**Comment by molant** [comment]
I've tried to this with the latest version in the master branch and published version on npm (0.14.1) and I can't reproduce it.

@melanierichards please try with the latest one and reopen if you still have issues. Thanks!
